docs/user-guide/common/beam-center-finder.ipynb

```diff
@@ -58,7 +58,10 @@
     "providers = list(\n",
     "    sans.providers + isis.providers + isis.data.providers + isis.sans2d.providers\n",
     ")\n",
-    "pipeline = sciline.Pipeline(providers, params=params)"
+    "pipeline = sciline.Pipeline(providers, params=params)\n",
```

Comment:
I guess some of the changes here clash with #136.
What is the merging order you had in mind?

Reply:
#136 will come first. I will update this PR once that is complete.